packages/codecs-strings/src/base64.ts

```diff
@@ -0,0 +1,52 @@
+import { combineCodec, Decoder, Encoder, mapDecoder, mapEncoder } from '@solana/codecs-core';
+
+import { getBaseXResliceDecoder, getBaseXResliceEncoder } from './baseX-reslice';
+
+const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/';
+
+/** Encodes strings in base64. */
+export const getBase64Encoder = (): Encoder<string> => {
+    if (__BROWSER__) {
+        return {
+            description: `base64`,
+            encode(value: string): Uint8Array {
+                try {
+                    const bytes = (atob as Window['atob'])(value)
+                        .split('')
+                        .map(c => c.charCodeAt(0));
+                    return new Uint8Array(bytes);
+                } catch (e) {
+                    // TODO: Coded error.
+                    throw new Error(`Expected a string of base 64, got [${value}].`);
+                }
+            },
+            fixedSize: null,
+            maxSize: null,
+        };
+    }
+
+    return mapEncoder(getBaseXResliceEncoder(alphabet, 6), (value: string): string => value.replace(/=/g, ''));
```

Comment:
Following up from the last PR.

* If we're in Node, surely we should use the (presumably) faster `Buffer.from('...', 'base64')` and `buffer.toString('base64')` right?
* You could always test `getBaseXResliceEncoder` separately?

What's the strategy for React Native here?

* Use the `getBaseXResliceEncoder` implementation?
* Get a `if (__REACTNATIVE__)` branch going?
* Do what you did for `text-encoding-impl` and make a `base64-impl` private package.

Reply:
Sounds good, I think it makes sense to:
- Use `Buffer` for `__NODEJS__`.
- Use `getBaseXResliceEncoder` for `__REACTNATIVE__`.
- Test `getBaseXResliceEncoder` separately.

I'll do that now.

Reply:
Done ✅